Jamfile

```diff
@@ -788,13 +788,24 @@ lib torrent
 
 headers = [ path.glob-tree include/libtorrent : *.hpp ] ;
 
-package.install install
+package.install install-torrent-lib
 	: <install-source-root>libtorrent
 	:
 	: torrent
 	: $(headers)
 	;
 
+local paths = [ package.paths torrent-rasterbar ] ;
+
+local datarootdir = [ $(paths).datarootdir ] ;
+
+install install-cmake
```

Comment:
I think you need to make this target *explicit*. which means it won't be built by default and you have to specify it on the command line to build it.

```
explicit install-cmake ;
```

Reply:
But I want to always install this file like the pkg-config file is installed.

Reply:
you want it to always be installed when someone runs `b2 install`, not just every time someone runs `b2`, right?

Currently the CI is failing because it's building multiple targets in the same command, both `darwin_ios` and `darwin_ios-sim`. That's not compatible with the install target, since it effectively "joins" all targets into the install location. The binaries would clobber each other, so b2 errors out instead

Reply:
Ah, thank you for the explanation! I had to add the "install" target to explicit as well.